packages/blade/src/components/Badge/__tests__/Badge.web.test.tsx

```diff
@@ -0,0 +1,149 @@
+import { Badge } from '../Badge';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import { InfoIcon } from '~components/Icons';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
```

Comment:
Why do we do this 🤔

Reply:
Because there's an error that's thrown in one of the test cases. The test case is to ensure that the error is thrown (when no text is passed) but if we don't mock it, it will throw an error in the jest process itself